pkg/endpoint/endpoint.go

```diff
@@ -62,14 +64,14 @@ func Listen(ctx context.Context, config Config) (ETCDConfig, error) {
 
 	listen := config.Listener
 	if listen == "" {
-		listen = KineSocket
+		listen = "KineSocket"
```

Comment:
This is not right - `KineSocket` is a variable, why did you quote it?
```go
KineSocket      = "unix://kine.sock"
```

Reply:
The best reason i can think of is a typo 🤦